learnEcl/2500-transform.md

```diff
@@ -2,3 +2,195 @@
 title: Transform
 slug: transform
 ---
+
+# TRANSFORM
+
+TRANSFORM functions are invoked by another function and make operations that must be performed on entire datasets. TRANSFORM functions start from row one and cover the entire dataset row by row. When defining a transform you need to tell the function what needs to be done on each field in the dataset by using input dataset fields or by creating new definitions for the fields.
```

Comment:
"invoked by another function and make operations" -- there is something off about that statement.

"TRANSFORM functions start from row one..." -- TRANSFORMS don't do that.  The function calling the TRANSFORM "starts from row one" or whatever, but the TRANSFORM itself does not care.

TRANSFORM's job is to create a *single* new record of a given type.  It may accept arguments or use a global context variable; it has the same limits as a regular function in that regard.